Analysis/Tutorials/src/histograms.cxx

```diff
@@ -78,11 +78,32 @@ struct CTask {
   }
 };
 
+struct DTask {
+  OutputObj<TList> list{"list"};
```

Comment:
Is there a particular motivation to still allow "untyped" ROOT containers in which objects of completely different nature are passed in? 
In modern terms this is essentially the same thing as std::vector<TObject*> - in this particular case probably just std::vector<TH1F*>.

Reply:
This is only a work around until we have the histogram registry which is in a few weeks. std::vector cannot be used as output at the moment with OutputObj to my understanding.